src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToStartHandlerTest.java

```diff
@@ -139,6 +140,41 @@ public void givenAppealWithTimeExtension_populateDirectionTypeDropdown() {
         assertEquals(4, listOptions.size());
     }
 
+    @Test
+    public void givenSpecificBenefitCodeAppeal_populateDirectionTypeDropdown() {
+        for (String benefitCode : BENEFIT_CODES_FOR_ISSUE_AND_SEND_TO_ADMIN) {
+            when(callback.getEvent()).thenReturn(EventType.DIRECTION_ISSUED);
+            sscsCaseData.setBenefitCode(benefitCode);
+
+            List<DynamicListItem> listOptions = new ArrayList<>();
+            listOptions.add(new DynamicListItem(APPEAL_TO_PROCEED.toString(), APPEAL_TO_PROCEED.getLabel()));
+            listOptions.add(new DynamicListItem(PROVIDE_INFORMATION.toString(), PROVIDE_INFORMATION.getLabel()));
+            listOptions.add(new DynamicListItem(ISSUE_AND_SEND_TO_ADMIN.toString(), ISSUE_AND_SEND_TO_ADMIN.getLabel()));
+
+            PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
+
+            DynamicList expected = new DynamicList(new DynamicListItem("", ""), listOptions);
+            assertEquals(expected, response.getData().getDirectionTypeDl());
+            assertEquals(3, listOptions.size());
```

Comment:
not sure what this assertion is trying to do, since listOptions is obviously going to have size 3, because we added 3 items to it in the test